extensions/wikia/Answers/Answers.php

```diff
@@ -23,31 +11,20 @@
 	
 );
 
-// FIXME: Migrate require_once's to inclues for performance reasons.
-require_once( dirname(__FILE__) . "/AnswersClass.php");
-require_once( dirname(__FILE__) . "/AttributionCache.class.php");
+$wgAutoloadClasses['Answers'] = __DIR__ . '/AnswersClass.php';
+$wgAutoloadClasses['AttributionCache'] = __DIR__ . '/AttributionCache.class.php';
 
 $wgExtensionMessagesFiles['Answers'] = dirname( __FILE__ ) . '/Answers.i18n.php';
 
 if(empty($wgAnswerHelperIDs)) {
 	$wgAnswerHelperIDs = array( 0 /* anonymous */, 1172427 /* Wikia User */ );
 }
 
-$wgHooks['NewRevisionFromEditComplete'][] = 'incAnswerStats';
-function incAnswerStats($article, $revision, $baseRevId) {
-	// This was only used for the social code which has been removed.
-	return true;
-}
-
-
 $wgAutoloadClasses['DefaultQuestion'] = dirname( __FILE__ ) . "/DefaultQuestion.php";
 $wgAutoloadClasses['PrefilledDefaultQuestion'] = dirname( __FILE__ ) . "/PrefilledDefaultQuestion.php";
 $wgAutoloadClasses['CreateQuestionPage'] = dirname( __FILE__ ) . "/SpecialCreateDefaultQuestionPage.php";
 $wgSpecialPages['CreateQuestionPage'] = 'CreateQuestionPage';
 
-$wgAutoloadClasses['GetQuestionWidget'] = dirname( __FILE__ ) . "/SpecialGetQuestionWidget.php";
-$wgSpecialPages['GetQuestionWidget'] = 'GetQuestionWidget';
```

Comment:
Why was `GetQuestionWidget` deleted?

Reply:
It was spectacularly broken with JS error since time immemorial